django_fido/models.py

```diff
@@ -91,3 +101,39 @@ def attestation(self) -> AttestationObject:
     def attestation(self, value: AttestationObject):
         self.attestation_data = base64.b64encode(value).decode('utf-8')
         self.credential_id_data = base64.b64encode(value.auth_data.credential_data.credential_id).decode('utf-8')
+
+    @property
+    def key_identification(self) -> Optional[str]:
+        """Return key identification.
+
+        Depending on protocol used to register the key, this can be:
+            aaid
+            aaguid
+            attestationCertificateKeyIdentifiers
+        """
+        # First test the presence of aaguid
+        if self.credential.aaguid != NULL_AAGUID:
+            return str(UUID(b2a_hex(self.credential.aaguid).decode()))
+        else:
+            # Get the certificate...
+            if 'x5c' in self.attestation.att_statement:
+                cert = self.attestation.att_statement['x5c'][0]
+                certificate = load_pem_x509_certificate(PEM_CERT_TEMPLATE.format(cert), default_backend())
+            else:
+                # ECDSAA attestation or self attestation?
+                return None
+
+    @property
+    def level(self) -> Optional[AuthLevel]:
+        """Return certification level."""
+        pass
+
+
+class AuthenticatorMetadata(models.Model):
+    """Stores information from metadata service."""
+
+    url = models.URLField()
+    identification = models.TextField()
```

Comment:
`identifier`, or just `aaguid` if it wouldn't be too confusing.

Reply:
It can be `aaid` or `attestationCertificateKeyIdentifiers` as well. Or we can do three mutually exclusive fields for each of these inputs.